.travis.yml

```diff
@@ -10,47 +10,6 @@ go:
     # test code to use newer parts of the standard library
     - 1.11.4
 
-env:
-    global:
-        # Should be encrypted or set as private travis variables (in travis settings):
-        # - AWS_S3_ACCESS_KEY_ID=''
-        # - AWS_S3_SECRET_ACCESS_KEY=''
-        # - GITHUB_RELEASE_TOKEN=''
-
-        # AWS S3 bucket name, need to have access setup.
-        # Can be overridden by travis private variable set in travis settings: AWS_BUCKET_OVERRIDE
-        - AWS_BUCKET_DEFAULT=mender-buildsystem
-
-        # AWS region.
-        # Can be overridden by travis private variable set in travis settings: AWS_REGION_OVERRIDE
-        - AWS_REGION_DEFAULT=eu-west-1
-
-        # AWS S3 ACL for build artifacts.
-        # Valid ACL values are: private, public_read, public_read_write, authenticated_read,
-        # bucket_owner_read, bucket_owner_full_control.
-        # Can be overridden by travis private variable set in travis settings: AWS_S3_ACL_OVERRIDE
-        - AWS_S3_ACL_DEFAULT=private
-
-        # Perform deployment steps if the build is performed for build: user/repo_name
-        # $TRAVIS_REPO_SLUG could be used if want to perform deploy on any repo.
-        # Currently perform deployment steps only of building main repository.
-        # Can be overridden by travis private variable set in travis settings: DEPLOY_REPO_OVERRIDE
-        - DEPLOY_REPO_DEFAULT=mendersoftware/mender
-
-        # Binary name to be uploaded to github release.
-        - GITHUB_RELEASE_BINARY=mender
```

Comment:
The GitHub releases seems to be broken also. I think we should fix it as there might be users looking for binaries directly on GitHub.

What do you think @kacf ?

Reply:
Yes, absolutely!

Reply:
Created [MEN-2545](https://tracker.mender.io/browse/MEN-2545)